src/core/lib/iomgr/lockfree_event.h

```diff
@@ -30,11 +30,13 @@ namespace grpc_core {
 class LockfreeEvent {
  public:
   LockfreeEvent();
-  ~LockfreeEvent();
 
   LockfreeEvent(const LockfreeEvent&) = delete;
   LockfreeEvent& operator=(const LockfreeEvent&) = delete;
 
+  void Init();
```

Comment:
I suggest renaming these to `InitEvent` and `DestroyEvent` - otherwise, this is confusing since `ManualConstructor` also has `Init` and `Destroy` methods.
(It was  very confusing to me to look at `fd_create` function and see `Init()` called two times :). One like this `foo.Init()` and the other like this: `foo->Init()` )

Reply:
Done :)